go/libraries/doltcore/doltdb/system_table.go

```diff
@@ -204,15 +204,17 @@ const (
 const (
 	// SchemasTableName is the name of the dolt schema fragment table
 	SchemasTableName = "dolt_schemas"
-
-	// Currently: `view`.
+	// SchemasTablesIdCol is an incrementing integer that represents the insertion index.
+	SchemasTablesIdCol = "id"
+	// Currently: `view` or `trigger`.
 	SchemasTablesTypeCol = "type"
-
-	// // The name of the database entity.
+	// The name of the database entity.
 	SchemasTablesNameCol = "name"
 	// The schema fragment associated with the database entity.
 	// For example, the SELECT statement for a CREATE VIEW.
 	SchemasTablesFragmentCol = "fragment"
+	// The name of the index that is on the table.
+	SchemasTablesIndexName = "fragment_name"
```

Comment:
We also need to update the next `const` block with new tags. However, the current tags in that block are currently incorrect/unused. See the comment in `sqle/schema_table.go` for details.